kvdb/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "kvdb"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
this is a breaking change since it's exposed in a public type `DBValue` :(

Reply:
yeah I'm reverting this, because we might as well do all the updating when we get to `kvdb-rocksdb 0.3`